src/linked-list.js

```diff
@@ -4,7 +4,33 @@ class LinkedList {
     this.tail = null;
     // Do not modify anything inside of the constructor
   }
-  
+  addToTail(item) {
+    const node = { value: item, next: null };
+    if (!this.head) {
+      this.head = node;
+    } else {
+      let currentNode = this.head;
+      while (currentNode.next) {
+        currentNode = currentNode.next;
+      }
+      currentNode.next = node;
+    }
+    this.tail = node;
+  }
+  removeHead() {
+    const removedValue = this.head.value;
+    const nextNode = this.head.next;
+    this.head = nextNode;
+    return removedValue;
+  }
+  contains(item) {
+    if (!this.head) return false;
+    let currentNode = this.head;
+    while (currentNode) {
```

Comment:
Well done! This is fun! There may be a few ways to simplify your methods but this works really well!